src/Features/CSharp/Portable/RemoveUnreachableCode/CSharpRemoveUnreachableCodeDiagnosticAnalyzer.cs

```diff
@@ -123,7 +123,7 @@ private void AnalyzeSemanticModel(SemanticModelAnalysisContext context)
                 if (fadeOutCode)
                 {
                     context.ReportDiagnostic(
-                        Diagnostic.Create(UnnecessaryWithoutSuggestionDescriptor, firstStatementLocation));
+                        DiagnosticHelper.CreateUnnecessaryLocationsWithoutSmartTag(Descriptor, firstStatementLocation));
```

Comment:
do we needto change this feature in this PR?  or can we do it in another?

Reply:
Another would be preferable. See https://github.com/dotnet/roslyn/pull/41854#discussion_r382925338.

Reply:
@CyrusNajmabadi Commits removed.